src/components/KTableData/KTableData.vue

```diff
@@ -19,7 +19,7 @@
     :hide-pagination="hidePagination || !showPagination"
     :hide-pagination-when-optional="false"
     :hide-toolbar="hideToolbar"
-    :loading="loading || fetcherIsLoading"
+    :loading="loading ?? fetcherIsLoading"
```

Comment:
https://github.com/Kong/konnect-ui-apps/blob/37492e0dcbc48d8376d4cc06b1fdc0132203f0ea/apps/gateway-manager/src/pages/GatewayLogsPage.vue#L74

It appears that it’s common practice across our tables in Konnect to initialize a `ref<boolean>` that defaults to `false` for many of our `KTableData` instances. 

Wouldn’t this usage pattern prevent the table’s built-in loading state from showing when it actually _does_ need to initially fetch the data when a component mounts?

Reply:
In the linked implementation, each time `fetcher` is called, it triggers `clearLoadingAndError`, which sets `isLoading` to `true`. The logic seems a bit convoluted, but I assume the behavior remains consistent. We can confirm this by testing it in a preview PR.

Actually, I believe we’re misusing `fetcherCacheKey` to trigger revalidation like this. Instead, we should directly include the fetcher parameters in the cache key.

For the current change, if users explicitly set the `loading` prop to `false`, we should consider respecting that.

Reply:
The behavior of the loading prop isn’t the core of the problem we’re facing. The key issue is that we haven’t exposed a “revalidate only” interface on the Table. The auto-incrementing cache key actually implies loading new data that hasn’t been loaded before, which, technically, is not accurate. Another viable solution is to `defineExpose` the `revalidate` method from `<KTableData>`, and also, for the scenario we’re currently encountering (polling), expose the `refreshInterval` configuration of SWRV as a prop. This way, we can trigger polling revalidation in a more declarative manner. WDYT?